src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  position: relative;
+  top: 50%;
+  margin: auto;
+  transform: translateY(25%);
+  width: (7 * $cell-size) + 7*$gap;
+  
+  &__field {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap;
+  
+    @each $day, $value in $days {
+      &--start-day--#{$day} :first-child {
+        margin-left: ($gap + $cell-size) * ($value - 1);
+      }
+    }
+    
+    @for $day from 29 through 31 {
+      &--month-length--28 :nth-child(n + #{$day + 1}) {
+        display: none;
+      }
+    }
+  }
+  
+  &__cell {
+    display: flex;
+    background-color: #eee;
+    box-sizing: border-box;
+    height: $cell-size;
+    width: $cell-size;
+    border: 1px solid black;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: $day-size;
+    transition: 0.5s;
+    cursor: pointer;
+  
+    @for $day from 1 through 31 {
+      &:nth-child(#{$day})::before {
+        content: "#{$day}";
+        margin: auto;
```

Comment:
move margin to &__cell as well

Reply:
Hello. When I move margin to &__cell, content jumps to the left corner (it is not at the centre anymore). Is it right?